src/processors/ipfsFile/errorProcessor.ts

```diff
@@ -29,7 +43,22 @@ export const ipfsFileErrorRetry: Processor = {
       numberOfRetries: (ipfsFile.numberOfRetries ?? 0) + 1,
       lastRetry: new Date()
     }));
-    await clients.db.upsert(Table.ipfsFiles, errorUpdates, 'url', undefined, true);
+    await clients.db.upsert(Table.ipfsFiles, errorUpdates, 'cid', undefined, true);
+  },
+  initialCursor: undefined
+}
+
+export const ipfsFileUrlErrorRetry: Processor = {
+  name: `ipfsFileUrlErrorRetry`,
+  trigger: ipfsFilesUrlsWithErrors,
+  processorFunction: async (input: IPFSFileUrl[], clients: Clients) => {
+    const errorUpdates: IPFSFileUrl[] = input.map((ipfsFileUrl) => ({
+      ...ipfsFileUrl,
+      error: undefined,
+      numberOfRetries: (ipfsFileUrl.numberOfRetries ?? 0) + 1,
+      lastRetry: new Date()
+    }));
+    await clients.db.upsert(Table.ipfsFilesUrls, errorUpdates, 'url', undefined, true);
```

Comment:
we really need to stop using upsert when we mean update - feels like this issue is just growing and we're spreading tech debt - maybe should fix the update function to actually work properly?

Reply:
Agreed. Problem is that our update doesn't support setting a field to undefined. Will add this to the changes on the other PR

Reply:
cool plz add to linear task